src/java.base/share/classes/jdk/internal/foreign/FunctionDescriptorImpl.java

```diff
@@ -0,0 +1,238 @@
+/*
+ *  Copyright (c) 2020, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
New source. No need for 2020

Reply:
The code in this class is basically copied from the original FunctionalDescription concrete class from 2020 so I think 2020 can stay here.